### PR TITLE
PLT-1800 Load server side locale from the config.json

### PR DESCRIPTION
**Pull request by enahum**
Once more, but this time lets create a test server

### CODE REVIEW

**Comment by crspeller** [comment]
Rebase please.

**Comment by coreyhulen** [comment]
+1

File: utils/i18n.go [hunk 1]
```diff
@@ -7,15 +7,17 @@ import (
 	"strings"
 
 	l4g "github.com/alecthomas/log4go"
-	"github.com/cloudfoundry/jibber_jabber"
+	//"github.com/cloudfoundry/jibber_jabber"
```
Comment (hmhealey) [inline utils/i18n.go]: Are we using this anywhere else now? If not, we should remove it completely

Comment (enahum) [inline utils/i18n.go]: Glide issues

```
$ glide rm github.com/cloudfoundry/jibber_jabber
[INFO] Preparing to remove 1 packages.
[INFO] Resolving imports
[INFO] Package gopkg.in/asn1-ber.v1 found in vendor/ folder
[INFO] Package github.com/mattermost/rsc/qr/coding found in vendor/ folder
[INFO] Package golang.org/x/image/bmp found in vendor/ folder
[INFO] Package golang.org/x/image/tiff found in vendor/ folder
[INFO] Package github.com/vaughan0/go-ini found in vendor/ folder
[INFO] Package github.com/golang/freetype/raster found in vendor/ folder
[INFO] Package github.com/golang/freetype/truetype found in vendor/ folder
[INFO] Package golang.org/x/image/font found in vendor/ folder
[INFO] Package golang.org/x/image/math/fixed found in vendor/ folder
[INFO] Package github.com/gorilla/context found in vendor/ folder
[INFO] Package github.com/nicksnyder/go-i18n/i18n/bundle found in vendor/ folder
[INFO] Package github.com/nicksnyder/go-i18n/i18n/language found in vendor/ folder
[INFO] Package github.com/nicksnyder/go-i18n/i18n/translation found in vendor/ folder
[INFO] Package github.com/rwcarlsen/goexif/tiff found in vendor/ folder
[INFO] Package github.com/golang/groupcache/lru found in vendor/ folder
[INFO] Package github.com/garyburd/redigo/redis found in vendor/ folder
[INFO] Package golang.org/x/crypto/blowfish found in vendor/ folder
[INFO] Package github.com/lib/pq/oid found in vendor/ folder
[INFO] Package golang.org/x/sys/unix found in vendor/ folder
[INFO] Package github.com/mattermost/rsc/gf256 found in vendor/ folder
[INFO] Package golang.org/x/image/tiff/lzw found in vendor/ folder
[INFO] Package gopkg.in/yaml.v2 found in vendor/ folder
[INFO] Package github.com/garyburd/redigo/internal found in vendor/ folder
[WARN] github.com/go-gorp/gorp appears to be a vendored package. Unable to set new version. Consider the '--update-vendored' flag.
```

@crspeller any thoughts on this?

Comment (enahum) [inline utils/i18n.go]: done

**Comment by hmhealey** [comment]
+1

**Comment by crspeller** [comment]
@enahum Please remove the dependency from the glide.lock file as well.

**Comment by enahum** [comment]
@crspeller done

**Pull request merged**